lib/widgets/Widgets.simba

```diff
@@ -135,12 +135,12 @@ end;
 
 function TReflectWidget.GetScrollX: Integer;
 begin
-  Result := Reflect.Smart.GetFieldInt(Self.Reference, Widget_ScrollX);
+  Result := Reflect.Smart.GetFieldInt(Self.Reference, Widget_RelativeX);
```

Comment:
Why make this naming wrong instead of just changing which accessor is used in the GetX/Y functions?

Also, haven't looked at the client in a while, but I think the x/y logic is outdated. It needs to take scroll and relative into account.